protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/fromconnectschema/FieldBuilder.java

```diff
@@ -91,4 +144,17 @@ private static void setProto3Optional(
         fieldBuilder.setProto3Optional(true);
         fieldBuilder.setOneofIndex(descriptorProtoBuilder.getOneofDeclCount() - 1);
     }
+
+    private static String getTypeName(String typeName) {
+        return typeName.startsWith(".") ? typeName : "." + typeName;
+    }
+
+    private static String toMapEntryName(String s) {
```

Comment:
This seems repeated. Can we have it one place?

Reply:
Will refactor.